test/e2e/e2e.go

```diff
@@ -29,26 +29,27 @@ import (
 	"k8s.io/ingress-nginx/test/e2e/framework"
 
 	// tests to run
-	_ "k8s.io/ingress-nginx/test/e2e/admission"
-	_ "k8s.io/ingress-nginx/test/e2e/annotations"
-	_ "k8s.io/ingress-nginx/test/e2e/annotations/modsecurity"
-	_ "k8s.io/ingress-nginx/test/e2e/dbg"
-	_ "k8s.io/ingress-nginx/test/e2e/defaultbackend"
-	_ "k8s.io/ingress-nginx/test/e2e/endpointslices"
-	_ "k8s.io/ingress-nginx/test/e2e/gracefulshutdown"
-	_ "k8s.io/ingress-nginx/test/e2e/ingress"
-	_ "k8s.io/ingress-nginx/test/e2e/leaks"
-	_ "k8s.io/ingress-nginx/test/e2e/loadbalance"
-	_ "k8s.io/ingress-nginx/test/e2e/lua"
-	_ "k8s.io/ingress-nginx/test/e2e/nginx"
-	_ "k8s.io/ingress-nginx/test/e2e/security"
-	_ "k8s.io/ingress-nginx/test/e2e/servicebackend"
-	_ "k8s.io/ingress-nginx/test/e2e/settings"
-	_ "k8s.io/ingress-nginx/test/e2e/settings/modsecurity"
-	_ "k8s.io/ingress-nginx/test/e2e/settings/ocsp"
-	_ "k8s.io/ingress-nginx/test/e2e/ssl"
-	_ "k8s.io/ingress-nginx/test/e2e/status"
-	_ "k8s.io/ingress-nginx/test/e2e/tcpudp"
+	// _ "k8s.io/ingress-nginx/test/e2e/admission"
```

Comment:
why are the other tasks commented out?

Reply:
We haven't finished writing e2e tests. Commented out the others for testing. Didn't mean to push this revision.